src/app/pages/BuyWithCreditCard/BuyWithCreditCard.tsx

```diff
@@ -123,6 +124,10 @@ export const BuyWithCreditCard: FC = () => {
     ? toLocalFormat(inputCurrency.maxAmount, { decimalPlaces: inputCurrency.precision })
     : undefined;
 
+  const outputAmountIsLegit = outputAmount && outputAmount > 0;
```

Comment:
Please try fixing the form validation schema in `src/app/pages/BuyWithCreditCard/hooks/use-buy-with-credit-card-form.ts` instead of adding one more variable. If it fails, use `isDefined` function here